conjure-postman-core/build.gradle

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+apply plugin: 'com.palantir.external-publish-application-dist'
```

Comment:
I think you want https://github.com/palantir/conjure-java/blob/3c66e208c149ac0d5604bf73240cb08a0e506f0f/conjure-java-core/build.gradle#L17